compiler/rustc_mir/src/interpret/eval_context.rs

```diff
@@ -683,14 +683,13 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         self.stack_mut().push(frame);
 
         // Make sure all the constants required by this frame evaluate successfully (post-monomorphization check).
-        for const_ in &body.required_consts {
-            let span = const_.span;
+        for (span, const_) in &body.required_consts {
```

Comment:
Oh right! we have required_consts. Perfect. I think we may be able to remove the span from Operand::Const after all

Reply:
Should we do this in a follow up PR? or do you want me to take some kind of action here?